src/Batch/Client/Tests/ObjectModel/BatchClientIntegrationTests/IntegrationTestUtilities/FileUploadUtilities.cs

```diff
@@ -0,0 +1,567 @@
+﻿using Microsoft.Azure.Batch;
+using Microsoft.Azure.Batch.FileStaging;
+using Microsoft.WindowsAzure.Storage;
+using Microsoft.WindowsAzure.Storage.Blob;
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Text;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace BatchClientIntegrationTests.IntegrationTestUtilities
```

Comment:
Permanent.  The Batch tests can't depend on the file staging DLL, because the file staging DLL depends on Batch.  So our Batch tests need either to be rewritten to use resource files instead of file staging, or they need a private file staging implementation sufficient just for their needs.